docs/examples.md

```diff
@@ -27,10 +27,10 @@ pipx --version  # prints pipx version
 pipx run pycowsay  --version  # prints pycowsay version
 pipx --python pythonX pycowsay
 pipx --spec pycowsay==2.0 pycowsay --version
-pipx --spec git+https://github.com/ambv/black.git black
-pipx --spec git+https://github.com/ambv/black.git@branch-name black
-pipx --spec git+https://github.com/ambv/black.git@git-hash black
-pipx --spec https://github.com/ambv/black/archive/18.9b0.zip black --help
+pipx --spec git+https://github.com/psf/black.git black
+pipx --spec git+https://github.com/psf/black.git@branch-name black
+pipx --spec git+https://github.com/psf/black.git@git-hash black
+pipx --spec https://github.com/psf/black/archive/18.9b0.zip black --help
```

Comment:
Not introduced in this PR but I just noticed these are all missing the `run` argument.

Reply:
True, and earlier examples in this block do not actually work (it won't work to put run options before `run`.) 

I'll fix them too.